templates/app-ts/tsconfig.json

```diff
@@ -1,22 +1,14 @@
 {
     "compilerOptions": {
-        "module": "commonjs",
+        "target": "ES2018",
+        "module": "CommonJS",
```

Comment:
In the fastify organization is used "commonjs" almost everywhere: it would be nice to have it also here. Both versions are schema compliant, though.

Reply:
I'll then change this for consistency with other repos.